README.md

```diff
@@ -1 +1,66 @@
-# Capstone-javascript
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone-javascript
+
+> an api based app to show lastest shows (movies , series)
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Webpack
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
```

Comment:
- [ ] [OPTIONAL] Please deploy your project and paste link on your `README` file!